content/community/events/cncf-kudo-webinar-2019-10.md

```diff
@@ -0,0 +1,10 @@
+---
+event: CNCF Webinar Series
+topic: Introducing the Kubernetes Universal Declarative Operator
+speaker: Gerred Dillon
+date: 2019-10-16
+location: Online
+url: https://www.cncf.io/webinars/introducing-the-kubernetes-universal-declarative-operator/
+---
+
+This is an online event, use [this link](https://zoom.us/webinar/register/1315705584423/WN_4aeXjmSoS3SONZxCSAwDxg) to register.
```

Comment:
```suggestion
This is an online event, use [this link](https://zoom.us/webinar/register/1315705584423/WN_4aeXjmSoS3SONZxCSAwDxg) to register.

<!-- more -->
```

Reply:
The html comment is needed in order to render the note as an excerpt on the overview page

Reply:
Thanks @meichstedt - have added that comment back in.